workflows/prognostic_c48_run/runtime/diagnostics/machine_learning.py

```diff
@@ -111,12 +111,15 @@ def rename_diagnostics(diags: Diagnostics):
         "net_heating",
         "column_integrated_dQu",
         "column_integrated_dQv",
+        "total_sky_downward_shortwave_flux_at_surface_override",
+        "total_sky_net_shortwave_flux_at_surface_override",
+        "total_sky_downward_longwave_flux_at_surface_override",
```

Comment:
These have been renamed in the merged version of VulcanClimateModeling/fv3gfs-wrapper#244:
```suggestion
        "override_for_time_adjusted_total_sky_downward_shortwave_flux_at_surface",
        "override_for_time_adjusted_total_sky_net_shortwave_flux_at_surface",
        "override_for_time_adjusted_total_sky_downward_longwave_flux_at_surface",
```